docs/architecture/cloud-native/resiliency.md

```diff
@@ -1,39 +1,41 @@
 ---
 title: Cloud-native resiliency
 description: Architecting Cloud Native .NET Apps for Azure | Cloud Native Resiliency
-ms.date: 06/30/2019
+author: robvet
+ms.date: 05/13/2020
 ---
+
 # Cloud-native resiliency
 
-[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+Resiliency is the ability of your system to react to failure and still remain functional. It's not about avoiding failure, but accepting failure as inevitable and crafting  your cloud-native services to respond to it. The end-goal is to return the application to a fully functioning state after a failure.
+
+Cloud-native systems take full advantage of the cloud computing model. Your services must know how to thrive in a dynamic, virtualized cloud environment. They'll make extensive use of Platform as a Service (PaaS) compute infrastructure and managed services. The caveat: Cloud infrastructure and networking are inherently unreliable.
 
-Resiliency is the ability of your system to react to failure and still remain functional. It isn't about avoiding failure. But it's about accepting that failure is inevitable in cloud-based systems and building your application to respond to it. The end-goal of resiliency is to return the application to a fully functioning state after a failure.
+Cloud vendors can detect and mitigate platform failure. Your service will be restarted, scale out and in, and distributed across nodes. Your services must be design services to detect these environmental scenarios and recover.  
```

Comment:
Rephrase this sentence - _Your services must be design services.._ - doesn't read right.

Reply:
Fixed.